### PR TITLE
[2015] 2015 Performance (Company) dashboard

### PR DESCRIPTION
**Pull request by andylolz**

### CODE REVIEW

**Comment by Floppy** [comment]
@andylolz give me the nod when this wants merging :)

**Comment by andylolz** [comment]
@Floppy this is ready for review now.

File: jobs/company_dashboard.rb [hunk 1]
```diff
@@ -78,6 +78,26 @@ def send_metric_with_targets event, data, options = {}
...
+  send_metric_with_targets '2015-trainers-trained', CompanyDashboard.trainers_trained(2015)
+  send_metric_with_targets '2015-Reach', CompanyDashboard.reach(2015)
+  send_metric_with_targets '2015-flagship-stories', CompanyDashboard.flagship_stories(2015)
+  send_metric_with_targets '2015-bookings', CompanyDashboard.bookings(2015), currency: "GBP"
+
+  legend = {
+    "network" => "Global Network",
+    "innovation" => "Innovation Unit",
+    "core" => "Core",
+  }
```
Comment (andylolz) [inline jobs/company_dashboard.rb]: This should probably point to the translations in en.yml instead

File: dashboards/company/2015.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<ul class="grid rows-2 cols-4" id="company">
+  <li class="row-1 col-1">
+    <div data-id="Lifetime-network-size" data-view="Number" data-unordered="true" data-title="<%= t 'company.network' %>" id="network-size" data-tooltip="<%= t 'company.tips.network' %>"></div>
```
Comment (andylolz) [inline dashboards/company/2015.erb]: So… Using Lifetime-network-size here matches the text in company.tips.network, but is almost certainly wrong.

I think what we probably want here is the cumulative network size up to 2015. If so, that will be easier to do once theodi/odi-metrics-tasks#37 is merged.

**Pull request merged**